pkg/monitor/backenddisruption/disruption_backend_sampler.go

```diff
@@ -282,15 +291,20 @@ func (b *BackendSampler) GetHTTPClient() (*http.Client, error) {
 			return
 		}
 
-		roundTripper, err := b.wrapWithAuth(http.RoundTripper(httpTransport))
+		var err error
+		rt := http.RoundTripper(httpTransport)
+		rt, err = b.wrapWithAuth(rt)
 		if err != nil {
 			b.httpClient = nil
 			b.httpClientErr = err
 			return
 		}
+		if len(b.userAgent) > 0 {
+			rt = transport.NewUserAgentRoundTripper(b.userAgent, rt)
```

Comment:
what is the default? should we add a meaningful default user agent?

Reply:
no default value per se, however, the sole user of the sampler `createAPIServerBackendSampler` sets a value.

Reply:
I don't think we should add a default user agent as it is up to the user that uses the sampler